[move-ide] Move IDE issue template for GitHub (#24008)
MystenLabs/sui: .github/ISSUE_TEMPLATE/ide-issue.md

---
name: Move IDE issue
about: Create a new report for issues encountered running Move IDE
title: Move IDE issue report'
labels: move-ide
assignees: 'awelc'
---

## IDE Setup

Fill this with the description of your setup:
* OS: <specify OS version>
* IDE/editor: <specify IDE/editor version>
* Move analyzer version: <specify `move-analyzer` version>

> When using Move VSCode [extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) provide, make sure you are using the most recent version of VSCode (or equivalent editor, such as Cursor) and of the Move VSCode extension. Additionally, provide the content of the **Move Client** tab accessible by selecting **View** -> **Output** from the main menu and opening the appropriate tab from the drop down menu. The output should look similar to the following:

Reviewer: ```suggestion
When using Move VSCode [extension](https://marketplace.visualstudio.com/items?itemName=mysten.move), make sure to use the most recent version of both VSCode (or equivalent editor, such as Cursor) and the Move VSCode extension. Additionally, provide the content of the **Move Client** tab. To access this data, select **View** -> **Output** from the main menu and open the appropriate tab from the drop down menu. The output should look similar to the following:
```

``` shell
INFO [10/21/2025, 10:16:14 AM]: mysten.move version 1.0.33
INFO [10/21/2025, 10:16:14 AM]: Creating extension context
INFO [10/21/2025, 10:16:14 AM]: configuration: {"auto-imports":true,"force-bundled":false,"inlay-hints":{"type":false,"param":false},"lint":"default","server":{"path":null},"sui":{"path":"/opt/homebrew/bin/sui"},"trace":{"server":"off"}}
INFO [10/21/2025, 10:16:14 AM]: Installing language server
INFO [10/21/2025, 10:16:14 AM]: Bundled version: 1.60.0
INFO [10/21/2025, 10:16:14 AM]: Standalone version: 1.60.0
INFO [10/21/2025, 10:16:14 AM]: CLI version: 1.58.2
INFO [10/21/2025, 10:16:14 AM]: Setting v1.60.0 of standalone move-analyzer installed at '~/.sui/bin/move-analyzer' as the highest one
INFO [10/21/2025, 10:16:14 AM]: Starting client...
```

When using a different editor (e.g., Vim, Emacs), provide its version and version of `move-analyzer` binary your editor is using. You can get `move-analyzer` version by running the following command:

``` shell
move-analyzer --version
```

## Steps to Reproduce Issue

> Fill this in with the concrete steps needed to reproduce the issue. Do your best here as it significantly increases the changes of the problem being addressed, but if the issues is not reproducible, skip this step and proceed to the following ones. When providing code in the reproduction steps, use the smallest buildable example that demonstrates the issue, removing any extraneous details.

Reviewer: ```suggestion
Provide the concrete steps needed to reproduce the issue. The more detail you provide, the better chance the problem can be addressed. If the issue is not reproducible, skip this step and proceed to the following ones. When providing code in the reproduction steps, use the smallest buildable example that demonstrates the issue, removing any extraneous details.
```

e.g.
1. Clone repository <repository>
1. Load file <file>.
2. Hover over language construct <construct> on line <line> in column <column>

## Expected Result

Specify what outcome you expected should have resulted, but didn't.

e.g.
Expected some on-hover information to appear when hovering over <construct> on line <line> in column <column>

## Actual Result

Specify what the actual unexpected outcome was.

e.g.

> No on-hover information was displayed when hovering over <construct> on line <line> in column <column>

Reviewer: ```suggestion
No on-hover information was displayed when hovering over `<construct>` on line `<line>` in column `<column>`.
```
Now that I look again, these are variables so may not make sense to use tics. Leaving though so I can hot that period.

Author: For what it's worth, I was inspired here with what's in the other templates where they do not use backticks

> ## Editor logs

Reviewer: ```suggestion
## Editor Logs
```
Consistency

Upon encountering and issue, capture `move-analyzer` logs that may help diagnosing the issue. 

> When using Move VScode extension, provide the content of the **Move** tab accessible by selecting **View** -> **Output** from the main menu and opening the appropriate tab from the drop down menu. Beginning of the log should look similar to the following:

Reviewer: ```suggestion
When using the Move VS Code extension, provide the content of the **Move** tab. To access this data, select **View** -> **Output** from the main menu and open the appropriate tab from the drop down menu. Beginning of the log should look similar to the following:
```
